frontend/src/pages/MediaPage.js

```diff
@@ -0,0 +1,89 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+import Typography from '@material-ui/core/Typography';
+import Grid from '@material-ui/core/Grid';
+import { withStyles } from '@material-ui/core/styles';
+
+import Page from '../components/Page';
+
+/**
+ * The media item page
+ */
+const MediaPage = ({ mediaItem, classes }) => (
+  <Page>
+    <section>
+      <Grid container spacing={16} className={ classes.gridContainer }>
+        <Grid item xs={12} className={ classes.playerWrapper } style={{paddingBottom:'56.25%'}}>
+          <iframe src={mediaItem.player_url} className={ classes.player } width="100%" height="100%" frameBorder="0" allowFullScreen>
+          </iframe>
+        </Grid>
+        <Grid item xs={12}>
+          <Typography variant="subheading" gutterBottom>
+            <a target='_blank' className={ classes.link } href={mediaItem.bestSource.url} download>
+              Download media
+            </a>
+            <a className={ classes.link } href={mediaItem.legacy_stats_url} style={{float: 'right'}}>
```

Comment:
Rather than having floats, could we put a nested grid?
```jsx
<Grid item xs={12}>
    <Grid container>
        <Grid item xs={6}>/* Download link */</Grid>
        <Grid item xs={6}>/* Stats link */</Grid>
    </Grid>
</Grid>
```